.github/workflows/backend-api-spec-update.yml

```diff
@@ -19,21 +19,27 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ref: 'infra/update-backend-api-spec'
+          fetch-depth: 0
 
       - name: Set up Java
         uses: actions/setup-java@v3
         with:
           java-version: 17
           distribution: temurin
+
+      - name: Set up Git
```

Comment:
how did it worked previously without this 🤦‍♂️

Reply:
I'm not sure, maybe `infra/update-backend-api-spec` doesn't have any changes\commits during git pull